src/validation.h

```diff
@@ -143,7 +143,6 @@ extern CCriticalSection cs_main;
 extern CBlockPolicyEstimator feeEstimator;
 extern CTxMemPool mempool;
 typedef std::unordered_map<uint256, CBlockIndex*, BlockHasher> BlockMap;
-extern BlockMap& mapBlockIndex GUARDED_BY(cs_main);
```

Comment:
Can't this be a `const &` instead? The same for `chainActive`.

Reply:
I mean instead of removing.

Reply:
the helpers are required by #15606 either way, so why not add them here already?